test/javascripts/unit/admin/modules/deselect_taxon_click_test.js

```diff
@@ -0,0 +1,92 @@
+
+
+module("DeselectTaxonClick", {
+    setup: function () {
+        var form =
```

Comment:
It probably makes sense to define these stub markup variables at the top of the file to prevent cluttering the startup function. The startup function can then just contain 

```
$('#qunit-fixture').append(form);
$('#qunit-fixture form').append(taxonomyTree);
$('#qunit-fixture form').append(breadcrumbs);
etc
```